tests/product/tests/conftest.py

```diff
@@ -84,6 +86,12 @@ def config_node_pre_test(cloudbeat_start_stop):
         if node.metadata.name != cloudbeat_agent.node_name:
             continue
         for temp_file in temp_file_list:
+            api_client.exec_command(
+                container_name=node.metadata.name,
+                command="mkdir",
+                param_value=os.path.dirname(temp_file),
```

Comment:
nit: Since we are already working with `pathlib.Path`, we can utilize it to get the parent directory of a file instead of importing `os.path`. By using `str(Path(temp_file).parent)`, we can directly obtain the parent directory path as a string.